packages/terra-date-picker/src/DatePicker.jsx

```diff
@@ -82,10 +82,14 @@ const propTypes = {
    */
   required: PropTypes.bool,
   /**
-   * An ISO 8601 string representation of the initial value to show in the date input.
-   * This prop name is derived from react-datepicker but is analogous to value in a form input field.
+   * An ISO 8601 string representation of the default value to show in the date input.
+   * This prop name is derived from react-datepicker but is analogous to defaultvalue in a form input field.
    */
   selectedDate: PropTypes.string,
+  /**
+   * The date value.
```

Comment:
I know this is already in the docs above but we should also indicate here that the value prop should only be used for a controlled date picker, that the `selectedDate` prop will have no effect if this is set, and that both `onChange` and `onChangeRaw` must be set as well.

Reply:
Updated: https://github.com/cerner/terra-framework/pull/738/commits/216700ce9829692129df8c0b3853af7a24c2e4f5